### ISSUE TITLE
[NEW] For Gmail

### ISSUE THREAD

**Issue by SerjSX** [tycrek/degoogle#330]
[//]: # ( Fill out to the best of your ability. )
[//]: # ( If an item is not applicable, feel free to remove the line. )

| Checklist |   |
| --------- | - |
| **I have read the guidelines in [CONTRIBUTING.md]** | yes|
| Include my name in [CONTRIBUTORS.md]                | yes|
| I am affiliated with this alternative               | no, not related with the developer. |

### Details

- **General**
  - **Alternative name**: FairEmail
  -  Required
  - **Alternative URL**: https://email.faircode.eu/
  - Required
  - **5/9/14 eyes**: unknown
  - **Description**: This email application for Android may not be popular, but it is focused on privacy. It's open source and free (with 1-time payment for additional feature unlocking, but it doesn't ruin the email experience), and focused on privacy. It's also recommended by many people. 
- **FLOSS**
  - Repository: https://github.com/M66B/FairEmail 
  - Active: yes
  - Download: 
  F-Droid: https://f-droid.org/en/packages/eu.faircode.email/
Playstore: https://play.google.com/store/apps/details?id=eu.faircode.email
  - Price: freemium (1-time payment, not monthly.)
- **Encryption**
  - HTTPS: yes
  - E2EE: i dont know
  - Modern Encryption: yes
- **Transparency**
  - 3rd-party audits: i dont know
- **Privacy**
  - Anonymous payments: no (only the one time payment)
  - Analytics: no 
  - Captcha: i dont know

[CONTRIBUTING.md]: ../blob/master/CONTRIBUTING.md
[CONTRIBUTORS.md]: ../blob/master/CONTRIBUTORS.md

**Comment by Glitchy-Tozier** [comment]
Unfortunately FairMail lacks tons of features though, if you don't pay for it  :/

**Comment by SerjSX** [comment]
> Unfortunately FairMail lacks tons of features though, if you don't pay for it :/

Yeah but still I'd put it on the list, these premium stuff wont affect the email experience.

**Comment by Glitchy-Tozier** [comment]
The main thing that annoys me is that it blocks searching the server for emails. Which _does_ affect the experience, at least in my opinion.

I agree that it should probably be listed though.

**Comment by SerjSX** [comment]
> The main thing that annoys me is that it blocks searching the server for emails. Which _does_ affect the experience, at least in my opinion.
> 
> I agree that it should probably be listed though.

At least open source and secure, i think people should use it instead. I'm sure there are more free based ones, but i also think this is worth adding in the list.

**Comment by virtadpt** [comment]
+1

The kinds of security measures all of us here seem to be interested in all have tradeoffs.  Trading off search for privacy and security is the most common and, I think, a reasonable one.

Exposing data for easier indexing and search means exposing data to the provider, which seems to be the one thing none of us are really comfortable with.

**Comment by Glitchy-Tozier** [comment]
@virtadpt Correct me if i'm wrong, but doesn't FairMail indeed _have_ a proper search if you pay for the full app?

To me it seemed to only do this as a way to get users to pay them money.

**Comment by virtadpt** [comment]
I don't know.

They need money to stay in business somehow.  The alternative is nobody pays, they have no revenue stream (ostensibly because they don't sell user information), and go out of business in a year or two.
